test/DistributionTest.js

```diff
@@ -0,0 +1,151 @@
+const DistributionContract = artifacts.require("./DistributionContract");
+const MyTokenn = artifacts.require("./MyTokenn");
+
+const {
+    ether,           // Big Number support
+    constants,    // Common constants, like the zero address and largest integers
+    expectEvent,  // Assertions for emitted events
+    expectRevert, // Assertions for transactions that should fail
+} = require('@openzeppelin/test-helpers');
+
+
+
+const chai = require("./setupchai.js");
+const BN = web3.utils.BN;
+const expect = chai.expect;
+
+contract("Distribution contract", async ([owner, acc2, acc3, acc4]) => {
```

Comment:
should have name of contract, not separate to two words. -> DistributionContract
also rename file name to DistributionContract.test.js -> use suffix .test for all test files in future